Python/Product/Analysis/SourceSpan.cs

```diff
@@ -60,16 +55,12 @@ public SourceSpan(int startLine, int startColumn, int endLine, int endColumn)
         /// <summary>
         /// The start location of the span.
         /// </summary>
-        public SourceLocation Start {
-            get { return _start; }
-        }
+        public SourceLocation Start { get; private set; }
 
         /// <summary>
         /// The end location of the span. Location of the first character behind the span.
         /// </summary>
-        public SourceLocation End {
-            get { return _end; }
-        }
+        public SourceLocation End { get; private set; }
```

Comment:
Same thing with mutable fields. You can use `SourceSpan(SourceLocation start, SourceLocation end)` constructor in `Union`.

Reply:
Yeah, there shouldn't be any need for the setter on these.

Reply:
This b/c I used `Range`. No need anymore.